### DISCUSSION TITLE
Need Help!

### DISCUSSION THREAD

**Discussion by SammyBoiii07** [community/community#124085] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

**Need some Readme file suggestions
For my profile**

**Comment by Atharvkote** [comment]
### Well! You can Do following things :

- Add Github Trophies 
- You can add Your Skill
      - Languages you know
      - Python Libraries 
      - Frameworks 
      - OSs
- You can Add your Github stats as well
- using repos Available on github you can design badge for your socials
Hope it helped!🙂
